prow/github/types.go

```diff
@@ -203,6 +203,7 @@ type IssueEvent struct {
 	Action IssueEventAction `json:"action"`
 	Issue  Issue            `json:"issue"`
 	Repo   Repo             `json:"repository"`
+	Label  Label            `json:"label"`
```

Comment:
Where is this used?

Reply:
It is not used anymore, I switched to just using `IssueEvent.Issue.Labels`.  I removed the commit that added this.